modules/cpdos/hho.py

```diff
@@ -19,28 +22,41 @@ def HHO(url, s, main_status_code, authent):
 
     while i < max_i:
         big_value = big_value + "0" * 50
-        h = {f"X-Oversized-Header-{i}":f"{big_value}"}
-        #print(h)
+        h = {f"X-Oversized-Header-{i}": f"{big_value}"}
+        logger.debug(h)
```

Comment:
Converted into debug statement instead of commented print

With the right configuration, these statements can stay and not be seen by users